index.js

```diff
@@ -28,23 +28,29 @@ const redacted =
 const redactHeaders =
   evolve({ authorization: redacted, cookie: redacted })
 
-const setup = (bugsnagClient, logger = paperplane.logger) => {
-  const notify = (err, opts) => {
-    const { req } = err
-
-    const metaData = {}
-    if (req) {
-      metaData.request = formatRequest(req)
-      delete err.req
-    }
+const isJsError = is(Error)
 
-    if (notifiable(err)) {
-      bugsnagClient.notify(err, mergeDeepRight(opts, { metaData }))
+const setup = (bugsnagClient, logger = paperplane.logger) => {
+  const notify = (err, opts = {}) => {
+    if (isJsError(err)) {
+      logger(err)
+
+      const { req } = err
+      const metaData = {}
+      if (req) {
+        metaData.request = formatRequest(req)
+        delete err.req
+      }
+
+      if (notifiable(err)) {
+        const options = mergeDeepRight({ metaData, severity: 'error' }, opts)
```

Comment:
I know we're struggling with under-reported errors... but should we really default to `error` at this level? Should `catchAndCry` specify its `severity: 'error'`?